<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArraySelectionWidget</class>
 <widget class="QTabWidget" name="ArraySelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="currentIndex">
   <number>3</number>
  </property>
  <widget class="QWidget" name="vertex_data">
   <attribute name="title">
    <string>Vertex DataContainer</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="horizontalSpacing">
     <number>4</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QListWidget" name="vertexVertexArrayList"/>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="vertexFieldArrayList"/>
    </item>
    <item row="1" column="2">
     <widget class="QListWidget" name="vertexEnsembleArrayList"/>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="vertexVertexCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Vertex Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="vertexFieldCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Field Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="vertexEnsembleCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Ensemble Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="edge_data">
   <attribute name="title">
    <string>Edge DataContainer</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="horizontalSpacing">
     <number>4</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="1" column="2">
     <widget class="QListWidget" name="edgeFieldArrayList"/>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="edgeEdgeArrayList"/>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="edgeVertexArrayList"/>
    </item>
    <item row="1" column="3">
     <widget class="QListWidget" name="edgeEnsembleArrayList"/>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="edgeEdgeCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Edge Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="edgeVertexCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Vertex Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="edgeFieldCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Field Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="edgeEnsembleCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Ensemble Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="surface_data">
   <attribute name="title">
    <string>Surface DataContainer</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="horizontalSpacing">
     <number>4</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="1" column="6">
     <widget class="QListWidget" name="surfaceEnsembleArrayList"/>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="surfaceVertexArraysCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Vertex Data</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QListWidget" name="surfaceEdgeArrayList"/>
    </item>
    <item row="0" column="6">
     <widget class="QCheckBox" name="surfaceEnsembleArraysCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Ensemble Data</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="surfaceEdgeArraysCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Edge Data</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QCheckBox" name="surfaceFieldArraysCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Field Data</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QListWidget" name="surfaceFieldArrayList"/>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="surfaceVertexArrayList"/>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="surfaceFaceArraysCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Face Data</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QListWidget" name="surfaceFaceArrayList"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="volume_data">
   <attribute name="title">
    <string>Volume DataContainer</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>4</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="3">
     <widget class="QCheckBox" name="volumeCellCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Cell Data</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QCheckBox" name="volumeFieldCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Field Data</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QListWidget" name="volumeFieldArrayList"/>
    </item>
    <item row="1" column="5">
     <widget class="QListWidget" name="volumeEnsembleArrayList"/>
    </item>
    <item row="0" column="5">
     <widget class="QCheckBox" name="volumeEnsembleCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Ensemble Data</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QListWidget" name="volumeCellArrayList"/>
    </item>
    <item row="1" column="2">
     <widget class="QListWidget" name="volumeFaceArrayList"/>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="volumeEdgeArrayList"/>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="volumeVertexArrayList"/>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="volumeFaceCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Face Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="volumeEdgeCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Edge Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="volumeVertexCB">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
font-size: 11pt;
}</string>
      </property>
      <property name="text">
       <string>Vertex Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
